### ISSUE TITLE
requestSingleInstanceLock() first paragraph of documentation is misleading

### ISSUE THREAD

**Issue by robatwilliams** [electron/electron#17188]
> This method makes your application a Single Instance Application - instead of allowing multiple instances of your app to run, this will ensure that only a single instance of your app is running, and other instances signal this instance and exit.

As the name suggests and the following paragraphs explain, this method obtains a lock which the developer can then use to make the application single instance. It doesn't in itself make the application a single instance one.

Perhaps a hangover from when this method used to be called `makeSingleInstance()` (but had the current behaviour) ?

**Comment by MarshallOfSound** [comment]
Hm I see your point, this could probably do with a bit of rewording.  Do you want to take a shot at that?

> Perhaps a hangover from when this method used to be called `makeSingleInstance()`

Even when it was `makeSingleInstance` it didn't **actually** make it single instance.  It just obtained a lock like the current API does, that's part of the reason why we renamed / refactored the API.

**Comment by robatwilliams** [comment]
Completed by #17304

**Comment by focusedMonk** [comment]
Another issue which I have noticed is, I have created two electron apps where I need to open only one instance of both the applications at a time.
I am using app.requestSingleInstanceLock() to obtain the lock for first instance. When I open app_1 and try to open app_2, app_2 fails to obtain the lock since app_1 has already obtained. This condition is right when I open the app_1 for the second time.
